OMCompiler/SimulationRuntime/c/simulation/solver/linearSystem.c

```diff
@@ -393,6 +389,18 @@ int freeLinearSystems(DATA *data, threadData_t *threadData)
       freeAnalyticJacobian(jacobian);
       /* Note: The Jacobian of data->simulationInfo itself will be free later. */
 
+    if (linsys[i].parDynamicData == NULL)
```

Comment:
Is `parDynamicData` itself deleted somewhere? It should probably be.

Reply:
Yes, in https://github.com/OpenModelica/OpenModelica/blob/master/OMCompiler/SimulationRuntime/c/simulation/solver/linearSystem.c#L317

Reply:
Okay. Just making sure. It should be set to null there btw. Idk the workflow but if you check for null here after it is deleted it might break.